components/patient-data/ui/src/main/resources/PhenoTips/PatientClass.xml

```diff
@@ -257,6 +257,19 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
     </date_of_death_entered>
+    <delivery_mode>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <name>delivery_mode</name>
+      <number>47</number>
+      <picker>0</picker>
+      <prettyName>Delivery mode</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
```

Comment:
Some of these have [HPO terms](http://www.human-phenotype-ontology.org/hpoweb/showterm?id=HP:0001787). We could use those instead, request new HPO terms for the ones that don't (primary/secondary CS as children of CS, NSVD, VBAC, Odon Device), and display it like _Global Age of Onset_.